src/components/AuthRoute.tsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import { Route, Redirect, RouteComponentProps } from "react-router-dom";
 import { useWebnative } from "../context/webnative";
+import { Feed } from "../utils/feed";
 
 interface Props {
   component: React.FC<RouteComponentProps>;
   path: string;
   exact?: boolean;
+  feed: Feed | null | undefined;
```

Comment:
Should probably pick null or undefined - do we need both?

Reply:
👍 yes I can't remember why that is there. I just changed it though and ts seems happy